redaxo/src/core/lib/Fragment/Enum/Size.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Redaxo\Core\Fragment\Enum;
+
+enum Size: string
```

Comment:
hier auch ein sehr allgemeiner name, der ggf. spezifischer sein sollte?

analog https://github.com/redaxo/redaxo/pull/5807#discussion_r1334942988

Reply:
oder ist "size" in shoelace ein allgemeiner begriff der überall nur genau diese 3 werte hat?

Reply:
shoelace hat die zwar pro component einzeln definiert, aber beim überfliegen gibt es scheinbar immer genau diese drei values.
https://shoelace.style/components/button#sizes
https://shoelace.style/components/radio#sizes

Reply:
Das ist ein Thema was @gharlan und ich schon mal diskutiert hatten. Am Anfang war es spezifischer und jetzt stellte sich heraus, dass man diese allgemeiner halten könnte.

Reply:
wenn sie in shoelace selbst je komponente sind sollten wir es hier auch machen, finde ich.

Reply:
Habe diese jetzt spezifischer gesetzt.

Reply:
danke - ich finde es macht so mehr sinn